README.md

```diff
@@ -124,16 +120,20 @@ Linux
 ```bash
 sudo rm -rf /tmp/localstack
 ```
+Docker&nbsp;<sup><em>4</em></sup>
```

Comment:
do you mean `Docker` here?  The whole file is about when running on windows

Reply:
> "_...do you mean Docker here?..._" 

At line `123`? Yes. I mean Docker there.

So, that "[_Deleting localstack data between runs_](https://github.com/boydingham/prisoner-offender-search/tree/boyd-DT-2209-search-running-in-docker#deleting-localstack-data-between-runs)" section is saying: "_If running `localstack` on Linux, do this...If running `localstack` on Mac, do the that...If running `localstack` in **Docker**, do the third_..."


> "..._**The whole file** is about when running on windows_..."

You don't mean the entire `README` file is about running on Windows? Sorry Peter. I'm not clear on what you are referring to by that one.